.circleci/config.yml

```diff
@@ -1747,6 +1752,23 @@ jobs:
       - store_test_results:
           path: test/test-results/junit.xml
 
+  test-integration-main:
```

Comment:
Minor, but if there is no alternate types of integration tests (like the development unit tests), could we just call this `test-integration`?